frontend/src/utils/HistoryStatus.ts

```diff
@@ -0,0 +1,25 @@
+const statusClass = (param: string) => {
+  let complete = false;
+  let current = false;
+  let invalid = false;
+  let disabled = false;
+  switch (param) {
+    case 'complete':
+      complete = true;
+      break;
+    case 'current':
+      current = true;
+      break;
+    case 'invalid':
+      invalid = true;
+      break;
+    case 'disabled':
+      invalid = true;
```

Comment:
possible typo here, should it be `disabled = true`?

Reply:
Yep, it should be `disabled` I'll fix it, thanks!

Also, I can't seem to respond properly the other comment, so I'll answer here. But yes, it is related to how carbon designed the component, even without the `current` prop the last item of the progress indicator is set to current if not said otherwise, and there is no `incomplete` prop to set, so it stays like that, I'll check with the design team if that's okay.

Reply:
Talked with the UX team, we will use the current prop on the last item.